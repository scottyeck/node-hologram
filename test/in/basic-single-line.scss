//doc
// ---
// title: Basic 1
// name: 01-basic
// category: testing
// ---
// 
// Hello, world!
//enddoc

.selector {
	test: pass;
}

//doc
// ---
// title: Basic 2
// name: 02-basic
// category: testing
// ---
// 
// Hello, world, again!
//enddoc