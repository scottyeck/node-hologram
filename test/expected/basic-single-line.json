[{
	"title": "Basic 1",
	"name": "01-basic",
	"contents": "<p>Hello, world!</p>",
	"category": "testing"
}, {
	"title": "Basic 2",
	"name": "02-basic",
	"contents": "<p>Hello, world, again!</p>",
	"category": "testing"
}]